cuda/Simulation.cu

```diff
@@ -26,16 +27,27 @@ unsigned long long run_event_based_simulation_baseline(Inputs in, SimulationData
         int nthreads = 256;
         int nblocks = ceil( (double) in.lookups / (double) nthreads);
 
-
-        xs_lookup_kernel_baseline<<<nblocks, nthreads>>>( in, GSD );
-        gpuErrchk( cudaPeekAtLastError() );
+	int nwarmups = in.num_iterations / 10;
```

Comment:
Specifying the warmup iterations at runtime seems like it'd be more robust than hard-coding a fixed ratio

Reply:
Will do